.github/actions/clickhouse/action.yml

```diff
@@ -96,11 +96,11 @@ runs:
       shell: bash
       run: |
         cd /data/jfs/
-        git clone https://github.com/sanwan/chdata.git
+        git clone https://github.com/sanwan/chdata.git || git clone https://github.com/sanwan/chdata.git
```

Comment:
When `git clone` fail in the middle, we need to remove the partial dir and try again.